Public/Sdk/Public/Deployment/operations.dsc

```diff
@@ -114,16 +114,14 @@ function flattenItem(
 
     if (isStaticDirectory(item)) {
         return flattenStaticDirectory(item, targetFolder, handleDuplicateFile, currentResult, provenance);
-    } 
+    }
 
     if (isFile(item)) {
         return flattenFile(item, targetFolder.combine(item.name), handleDuplicateFile, currentResult, provenance);
     }
 
     if (isDeployable(item)) {
-        // TODO: TEMPORARY HACK: ['deploy'] just a little hack until I get an LKG using the NugetSpecGenerator.cs (ie, next build)

```

Comment:
I guess this "next build" took quite a while to produce :)

Reply:
yeah, I dropped the ball and forgot about it. Thanks for catching & fixing Kristijan!

---
In reply to: [282554031](https://github.com/microsoft/BuildXL/pull/281#discussion_r282554031) [](ancestors = 282554031)